tests/yam/validate/validate_migration_logs.pm

```diff
@@ -21,10 +21,16 @@ sub run {
     script_run("tar czvf /tmp/udev_rules.tar.gz /etc/udev/rules.d/*", {timeout => 60});
     upload_logs("/tmp/udev_rules.tar.gz", failok => 1);
 
+    # fatal migration errors that lead to failed test run
     if (script_run("cat /var/log/distro_migration.log | grep -i -E \"migration failed|aborting migration\"") == 0) {
         record_info("Migration failed", script_output("cat /var/log/distro_migration.log | grep -i -E \"migration failed|aborting migration\" -B50"), result => 'fail');
         die("Migration failed");
     }
+
+    # other non-fatal errors in migration log
+    if (script_run("cat /var/log/distro_migration.log | grep -iE \"failed|failure|error\" | grep -ivE \"\\-errors?|errors?\\-|error=''\"") == 0) {
```

Comment:
Could we avoid here running twice `cat /var/log/distro_migration.log | grep -ivE \"\\-errors?|errors?\\-|error=''\"` I guess we could save the result and pass it to the record info to further filtering.

Reply:
I think it would be better to have this. It is not complex. More info and better preliminary filter.

Reply:
I updated the code a bit. How do you think ? @jknphy

Reply:
thanks for the change, I like it more.